packages/components/src/ListView/ListView.component.js

```diff
@@ -36,13 +37,12 @@ function ListView(props) {
 ListView.displayName = 'ListView';
 
 ListView.propTypes = {
+	displayMode: PropTypes.string,
 	items: PropTypes.arrayOf(PropTypes.object),
-	t: PropTypes.func,
 };
 
 ListView.defaultProps = {
 	items: [],
-	isSwitchBox: false,
```

Comment:
This was not necessary, as it's only passed down to Item that redefine the default props